107/src/test/java/org/ehcache/jsr107/ConfigurationMergerTest.java

```diff
@@ -273,6 +278,53 @@ public void jsr107LoaderInitAlways() {
     assertThat(configHolder.useEhcacheLoaderWriter, is(false));
   }
 
+  @Test
+  public void jsr107DefaultEh107IdentityCopierForImmutableTypes() {
+    XmlConfiguration xmlConfiguration = new XmlConfiguration(getClass().getResource("/ehcache-107-default-copiers.xml"));
+    final DefaultJsr107Service jsr107Service = new DefaultJsr107Service(ServiceLocator.findSingletonAmongst(Jsr107Configuration.class, xmlConfiguration.getServiceCreationConfigurations().toArray()));
+    merger = new ConfigurationMerger(xmlConfiguration, jsr107Service, mock(Eh107CacheLoaderWriterProvider.class));
```

Comment:
Not true, with the code changes you have done, unless I missed something, there will be a smarter copier installed if you create a cache with immutable types purely in code using JSR-107 API. That is not tested for example.
And the code change you did in the configuration merger takes different path depending on the existence of default copier configurations at the CM level.